modules/subchannel/src/problems/QuadSubChannel1PhaseProblem.C

```diff
@@ -426,26 +426,38 @@ QuadSubChannel1PhaseProblem::computeBeta(unsigned int i_gap, unsigned int iz)
 Real
 QuadSubChannel1PhaseProblem::computeAddedHeatPin(unsigned int i_ch, unsigned int iz)
```

Comment:
the SCMPower  postprocessor could just call this routine btw to make the integral over pins

this would make it very consistent imo